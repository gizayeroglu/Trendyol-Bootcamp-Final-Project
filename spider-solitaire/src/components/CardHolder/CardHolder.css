.card-holder-container{
  display: flex;
  flex-direction: column;
} 