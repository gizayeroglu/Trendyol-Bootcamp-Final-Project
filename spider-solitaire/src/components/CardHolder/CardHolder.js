import './CardHolder.css';
import React from 'react';
import Card from '../Card/Card';

function CardHolder({ cards = [] }){
    return(
        cards.map((card, index) => {
          return(
            <>
              <Card key={index} content={card} />
            </>
          )
        })
    )
}

export default CardHolder;