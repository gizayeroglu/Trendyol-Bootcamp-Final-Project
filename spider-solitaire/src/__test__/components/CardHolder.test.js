import React from 'react';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { configure, shallow, mount } from 'enzyme';

import CardHolder from '../../components/CardHolder/CardHolder';

configure({ adapter: new Adapter() });

describe('Card Holder component tests', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CardHolder />);
  });

  it('should render the card holder component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
