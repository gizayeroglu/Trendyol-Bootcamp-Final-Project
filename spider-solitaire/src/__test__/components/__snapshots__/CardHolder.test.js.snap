// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Holder component tests should render 1`] = `ShallowWrapper {}`;
